### PR TITLE
Add unload event deprecation.

### PR DESCRIPTION
**Pull request by mwjacksonmsft**
Adds documentation to describe the unload event deprecation happening. 

AB#46367387

### CODE REVIEW

**Review by captainbrosset — APPROVED** [review]
Looks good to me. Flagging Michael for the final review/edit pass.

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -53,6 +53,7 @@ This table lists:
...
 | Removal of mutation events | v127 | | Removes support for mutation events in Chromium. Use the [MutationObserver](https://developer.mozilla.org/docs/Web/API/MutationObserver) API instead.  See [Intent to Deprecate: Mutation Events](https://groups.google.com/a/chromium.org/g/blink-dev/c/qDsKRU-cQ_4/m/isA1mZ_aAAAJ). |
+| Deprecate unload event | Future release (TBD) | | Introduces a new Permission-Policy to allow creating unload event listeners. Initially the default policy will be allow, but will gradually migrate the default policy to deny such that unload handlers stop firing on pages unless a page explicitly opts in to re-enable them.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see [Intent to Deprecate: Deprecate unload event](https://groups.google.com/a/chromium.org/g/blink-dev/c/dvusqw9-IhI/m/SBkm_u1RAQAJ). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:56]:
```suggestion
| Deprecate unload event | Future release (TBD) | | Introduces a new Permission-Policy to allow creating unload event listeners. Initially, the default policy will be `allow`, but the default policy will gradually be migrated to `deny`, such that unload handlers stop firing on pages, unless a page explicitly opts in to re-enable them.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see [Intent to Deprecate: Deprecate unload event](https://groups.google.com/a/chromium.org/g/blink-dev/c/dvusqw9-IhI/m/SBkm_u1RAQAJ). |
```
* allow - differentiate from regular words by format.
* gradually - grammar.
* deny - differentiate from regular words by format.

**Review by mikehoffms — CHANGES_REQUESTED** [review]
Writer/Editor pass

**Comment by mwjacksonmsft** [comment]
Thanks @mikehoffms, can you PTAL at the new revision?

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**